ncl/ncl_entries/sin_trigonometry.ipynb

```diff
@@ -0,0 +1,282 @@
+{
```

Comment:
<p>This might be picky of me, but I think we should call this page either Sine or Trigonometry (with the assumption that we'd expand the section later or add a WIP admonition or something if we wanted to publish with just the sin functions).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NCAR/geocat-applications/pull/40/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
Yeah, this is probably more of a design / discussion point, but I think we may want to scale up the category here to at least trigonometry if not something like mathematical functions and keep it pretty light on description.

Reply:
Since they are all using numpy/math I combined them into the most recent notebook with an overview at the top